src/controllers/update/overseas.entity.change.controller.ts

```diff
@@ -47,7 +48,10 @@ export const post = async (req: Request, resp: Response, next: NextFunction) =>
       resetDataForChange(appData);
       redirectUrl = config.WHO_IS_MAKING_UPDATE_URL;
     }
+
+    setExtraData(session, appData);
```

Comment:
Will this not affect register journey if its moved here?

Reply:
This controller is only for update journey. We need to set everything in the Web session so moved that down.